src/Directive.php

```diff
@@ -3,7 +3,7 @@
 namespace Mazedlx\FeaturePolicy;
 
 use Mazedlx\FeaturePolicy\Directives\DefaultDirective;
-use Mazedlx\FeaturePolicy\Exceptions\InvalidDirective;
+use Mazedlx\FeaturePolicy\Exceptions\UnknownPermissionGroupException;
```

Comment:
this relates to the different types of permissions. (default, proposed, experimental & retired)